### PR TITLE
Prepare 2.0.2

### PR DESCRIPTION
**Pull request by MetRonnie**
It's my first time so hopefully I've done it right

### CODE REVIEW

File: .codecov.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
 # ----------------------------------------------------------------------------
-# Copyright (C) 2013-2019 British Crown (Met Office) & Contributors.
+# Copyright (C) British Crown (Met Office) & Contributors.
 #
```
Comment (MetRonnie) [inline .codecov.yml:4]: Am I right that we are no longer including the year?

File: CHANGES.md [hunk 2]
```diff
@@ -7,12 +7,16 @@ for a full listing of issues for each release.
 
 ## isodatetime 2.0.2 (Upcoming, 2020)
```
Comment (MetRonnie) [inline CHANGES.md:9]: Will need to update the date on release day

File: .codecov.yml (hunk 1 above)
Comment (kinow) [reply .codecov.yml:4]: I think so. Only in one place now I think (License file? Can't recall)

Comment (oliver-sanders) [reply .codecov.yml:4]: Thanks for doing this, very observant.

You can follow the model of Cylc Flow, just add a line in the `README.md`.

See the Cylc Flow PR here: https://github.com/cylc/cylc-flow/pull/3583/files

Comment (MetRonnie) [reply .codecov.yml:4]: Added, let me know if it should be moved up or down

**Comment by oliver-sanders** [comment]
Will need to bump the `__version__`.

File: .codecov.yml (hunk 1 above)
Comment (oliver-sanders) [reply .codecov.yml:4]: Looks fine to me.

**Comment by MetRonnie** [comment]
> Will need to bump the `__version__`.

Isn't that done? https://github.com/metomi/isodatetime/pull/169/files#diff-183a38a2f9f46f483668085b34665c9c

**Comment by oliver-sanders** [comment]
Sorry, couldn't find it when I looked.

**Comment by MetRonnie** [comment]
In that case I take it I don't need to wait for Tim?

**Comment by oliver-sanders** [comment]
Two reviews sufficient. Go for it.

**Pull request merged**